xcodeproj/internal/xcschemes/xcschemes.bzl

```diff
@@ -1179,10 +1179,40 @@ Address Sanitizer cannot be used together with Thread Sanitizer.
         ),
     )
 
+def _autogeneration_config(scheme_name_exclude_patterns = None):
+    """Dictionary containing scheme auto-generation configurations
+
+    Args:
+        scheme_name_exclude_patterns: A list containg regex patterns used to skip
+            creating matching schemes. These exclusion patterns do not apply to custom
+            schemes.
+
+            Example:
+
+            xcodeproj(
+                ...
+                scheme_name_exclude_patterns = xcschemes.autogeneration_config(
+                    scheme_name_exclude_patterns = [
+                        ".*somePattern.*",
+                        "^anotherPattern.*",
+                    ],
+                )
+            )
```

Comment:
I believe you should be able to make this embedded in a `starlark` code block, so it renders correctly in the docs.

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/3027/commits/3ae150b6e816a42f24a04f31e26ece8c0cadfcc0